src/main/java/com/commercetools/service/paypalPlus/impl/PaypalPlusPaymentServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package com.commercetools.service.paypalPlus.impl;
+
+import com.commercetools.exception.PaypalPlusServiceException;
+import com.commercetools.service.paypalPlus.PaypalPlusPaymentService;
+import com.paypal.api.payments.Payment;
+import com.paypal.base.rest.APIContext;
+import com.paypal.base.rest.PayPalRESTException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.Nonnull;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * <b>Note:</b> since {@link APIContext} caches <i>PayPal-Request-Id</i> one should not reuse the same instance of
+ * {@link PaypalPlusPaymentServiceImpl} for multiple Paypal requests.
+ */
+@Service
+@Scope("prototype")
+public class PaypalPlusPaymentServiceImpl extends BasePaypalPlusService implements PaypalPlusPaymentService {
+
+    @Autowired
+    public PaypalPlusPaymentServiceImpl(@Nonnull APIContext paypalPlusApiContext) {
+        super(paypalPlusApiContext);
+    }
+
+    @Override
+    public CompletionStage<Payment> create(@Nonnull Payment payment) {
+        return createPaymentStage(payment);
```

Comment:
This method implements 2 features:
  1. Creates asynchronous Completion stage instead of default Paypal Plus blocking operation
  2. Wraps **checked** `PayPalRESTException` to unchecked `PaypalPlusServiceException` so the method signature matches CompletionStage requirements (there are not allowed checked exceptions)

I've extended the method comment: c017c429e450d03b748f432b45f8ca419242b824